spec_analysis/jw8_redshift_fitting_solutions.ipynb

```diff
@@ -25,7 +25,9 @@
     "    - Get the library of spectra\n",
```

Comment:
<p>Is the commented out 1-100 range necessary to keep?&nbsp;Is running a smaller portion just for a quick-running notebook or for performance reasons?</p><p> </p><p>Optional: could replace the loop above with something like the following:</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false"># move to imports at top of notebook
from glob import glob&nbsp;&nbsp;
 
# could define earlier and and reuse for previous cases where only loading a single spectrum
def load_spectrum(filename):
&nbsp;&nbsp;file_contents = ascii.read(filename)
&nbsp;&nbsp;return Spectrum1D(spectral_axis=template['col1']/1.E4*u.um, flux=template['col2']*spec_unit)
 
# glob takes any file matching patters, so could also do "*.dat" and then [30::40] to slice a subset of that
templatelist = [load_spectrum(filename) for filename in sorted(glob(templatedir+"0003?.dat"))]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Thank you for catching this. Removed the option for the 30:40 slice.